pkg/pebinaryexecutor/pebinary.go

```diff
@@ -220,11 +220,16 @@ func (p *PEBinaryConfig) KubeProxy(ctx context.Context, args []string) error {
 
 	logrus.Infof("Running RKE2 kube-proxy %s", args)
 	go func() {
+		outputFile, err := os.Create(filepath.Join(p.DataDir, "agent", "logs", "kube-proxy.log"))
```

Comment:
I would recommend reusing or at least duplicating the code that we use to capture kubelet logs using lumberjack, which handles rotating and pruning logs. Otherwise this one log file may get very large.

https://github.com/rancher/rke2/blob/f2396b6ea0a97b568a46b0fd754bac042e4ce854/pkg/logging/logging.go#L38-L40

Reply:
good idea! I was actually wondering if these logs could get too large. I was not aware of this lumberjack library